bin/subscribeUsersToPersonalizedDigest.ts

```diff
@@ -0,0 +1,46 @@
+import '../src/config';
+import createOrGetConnection from '../src/db';
+
+(async (): Promise<void> => {
+  const createdFromArgument = process.argv[2];
+  const createdToArgument = process.argv[3];
+
+  if (!createdFromArgument || !createdToArgument) {
+    throw new Error('createdFrom and createdTo arguments are required');
+  }
+
+  const createdFromDate = new Date(createdFromArgument);
+
+  if (Number.isNaN(createdFromDate.getTime())) {
+    throw new Error(
+      'createdFromDate argument is invalid, format should be ISO 6801',
+    );
+  }
+
+  const createdToDate = new Date(createdToArgument);
+
+  if (Number.isNaN(createdToDate.getTime())) {
+    throw new Error(
+      'createdToDate argument is invalid, format should be ISO 6801',
+    );
+  }
+
+  if (createdFromDate > createdToDate) {
+    throw new Error(
+      'createdFrom argument should be less than createdTo argument',
+    );
+  }
+
+  const con = await createOrGetConnection();
+
+  await con.transaction(async (manager) => {
+    await manager.query(`
```

Comment:
Since there's no CDC on this table we can just run the sql query without batching, can't we?

Reply:
Yeah, waited to see what you will say, ~300k rows in one query should be ok in one go. Although down the line script gives us more flexibility I think.